frontend/src/employees/EmployeeForm.js

```diff
@@ -1,52 +1,42 @@
 import React from 'react'
 import { Formik, Form } from 'formik'
-import * as yup from 'yup'
 import { Link } from 'react-router-dom'
 import Input from '../Input'
 import Button from '../Button'
 import PopUp from '../PopUp'
-
-const schema = yup.object().shape({
-  lastName: yup.string().required('A vezetéknév kötelező!'),
-  firstName: yup.string().required('A keresztnév kötelező!'),
-  email: yup
-    .string()
-    .required('Az email kötelező!')
-    .email('Nem megfelelő email cím!'),
-  password: yup
-    .string()
-    .required('A jelszó kötelező!')
-    .matches(
-      /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,})/,
-      'A jelszó minimum 8 karakter hosszú, tartalmaznia kell kis- és nagybetűt, valamint számot!'
-    ),
-  confirmPassword: yup
-    .string()
-    .required('Add meg a jelszót még egyszer!')
-    .oneOf([yup.ref('password')], 'A két jelszó nem egyezik meg!'),
-})
+import { schema, regSchema } from './ValidationEmplyee'
 
 function CreateEmployeeForm({
   sent,
   handleClick,
   popUpMessage,
   sendData,
   title,
+  user,
+  registration,
 }) {
+  const initialValuesCreate = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+    confirmPassword: '',
+  }
+  const initialValuesEdit = {
+    firstName: user?.firstName || '',
+    lastName: user?.lastName || '',
+    email: user?.email || '',
+  }
   return (
     <div className="container my-5">
       <div className="row justify-content-center">
         <div className="col-12">
           {sent && <PopUp handleClick={handleClick} body={popUpMessage} />}
           <Formik
-            initialValues={{
-              firstName: '',
-              lastName: '',
-              email: '',
-              password: '',
-              confirmPassword: '',
-            }}
-            validationSchema={schema}
+            initialValues={
+              registration ? initialValuesEdit : initialValuesCreate
```

Comment:
Lehet jobb lenne egy initialValues és ott ternary-zni a két érték között.

Reply:
Ezt nem értem. a registration arra kell h regisztráció e vagy sem és aszerint adja meg h mi legyen ott.

Reply:
const InitialValues = registration ? initialValuesEdit : initialValuesCreate

Vagy így gondoltad? Külön?

Reply:
Így is lehet, vagy akár rögtön a literal-okkal: const InitialValues = registration ? { ... // egyik } : { ... // másik }